.github/CODEOWNERS

```diff
@@ -82,6 +82,9 @@
 # PRLabel: %Synapse
 /sdk/synapse/                                                        @aim-for-better @idear1203
 
+# PRLabel: %EngSys
+/sdk/template/                                                       @Azure/azure-sdk-eng
```

Comment:
This isn't currently supported. I don't have a "detect whether the id is a user or slug" right now. Please add real users. I'll file a follow-up issue to add the capability you're looking for.

Reply:
Please can you go ahead and approve this. It should be fine for the automatic version increment. Can you also review https://github.com/Azure/azure-sdk-tools/pull/981 where I updated code owners to detect ID vs slug

Reply:
Do we know what the script does today? Won't it just warn like it would if there is user that is not recognized or doesn't have access?